README.md

```diff
@@ -45,7 +45,7 @@ A few installation options available.
 *   [Installing Apache PredictionIO from
     Binary/Source](http://predictionio.apache.org/install/install-sourcecode/)
 *   [Installing Apache PredictionIO with
-    Docker](http://predictionio.apache.org/community/projects/#docker-images)
+    Docker](http://predictionio.apache.org/install/install-docker/)
     (community contributed)
```

Comment:
Thanks for catching this @mmoreram, I forgot to modify README while changing the docker doc.
Can you also remove the `community contributed` tag?
Cus the new link that you are pointing to is not community projects.

Reply:
You're welcome :)
Updated!
Happy to help!